rcongui/src/components/PlayerView/playerActions.js

```diff
@@ -143,7 +143,7 @@ class ReasonDialog extends React.Component {
             label="Comment"
             variant="outlined"
             margin="dense"
-            helperText="A comment that will NOT be displayed to the player"
+            helperText="Extra info attached to the players profile regarding the `Watchlist`."
```

Comment:
Should remove the whatchlist bit here or use 
```
helperText={`Extra info attached to the players profile regarding the ${open.actionType}.`}
```

Reply:
Just opened the pull to keep it in mind, i was not sure this was even the correct text :D